node/src/main/scala/com/wavesplatform/database/LevelDBWriter.scala

```diff
@@ -172,14 +171,18 @@ class LevelDBWriter(writableDB: DB, spendableBalanceChanged: Observer[(Address,
     Map.empty
   )
 
-  private def loadPortfolio(db: ReadOnlyDB, addressId: BigInt) = loadLposPortfolio(db, addressId).copy(
+  private def loadPortfolio(db: ReadOnlyDB, addressId: BigInt, excludeNFT: Boolean) = loadLposPortfolio(db, addressId).copy(
     assets = (for {
-      asset <- db.get(Keys.assetList(addressId))
+      issuedAsset <- db.get(Keys.assetList(addressId))
+      asset <- transactionInfo(issuedAsset.id).collect {
```

Comment:
This will seriously slow down portfolio retrieval when NFT feature has not been activated yet. I suggest creating two distinct branches, one for when NFT is activated, and another for when it's not.